code/modules/cargo/packs/science.dm

```diff
@@ -111,8 +111,9 @@
 /datum/supply_pack/science/nuke_b_gone
 	name = "Nuke Defusal Kit"
 	desc = "Contains set of tools to defuse a nuke."
-	cost = 7500 //Usefull for traitors/nukies that fucked up
+	cost = 7500 //Useful for traitors/nukies that fucked up
 	dangerous = TRUE
+	contraband = TRUE
 	DropPodOnly = TRUE
```

Comment:
Why have this emag only and drop pod?

Reply:
As a Cargo player, I don't believe drop pod consoles are a legitimate barrier to access due to how easily they can be set up.

If a maintainer disagrees, I can remove the drop pod restriction later.

Reply:
We discussed this further over Discord while I was at work.